### PR TITLE
[Netmanager]Improving on the Organization Selection

### PR DESCRIPTION
**Pull request by danielmarv**
#### Summary of Change

- I have made some changes on the organization dropdown whereby now it can help in the selection
of an organization from many.

- I have used Mui StyledMenu.

- I have also used a a tooltip to show a header "Organization" on hover of the Organization button 

#### Tasks:
- To accommodate plenty of data 

#### Status of maturity

- [x] I've tested this locally
- [x] I consider this code done

#### What are the relevant tickets?

- [AN-458](https://airqoteam.atlassian.net/browse/AN-458)

#### Screenshots (optional)

[AN-458]: https://airqoteam.atlassian.net/browse/AN-458?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZj
dan
UiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
dan1

### CODE REVIEW

**Review by Codebmk — CHANGES_REQUESTED** [review]
Replace "Select Network" with "Select Organisation"
image

File: netmanager/src/views/layouts/common/Sidebar/components/NetworkDropdown.js [hunk 1]
```diff
@@ -11,110 +19,101 @@ import {
...
+  const [openDialog, setOpenDialog] = useState(false);
+  const classes = useStyles();
 
   useEffect(() => {
     const activeNetwork = JSON.parse(localStorage.getItem('activeNetwork'));
     if (activeNetwork) {
       setSelectedNetwork(activeNetwork);
-    } else {
+    } else if (userNetworks.length > 0) {
       setSelectedNetwork(userNetworks[0]);
```
Comment (Codebmk) [inline netmanager/src/views/layouts/common/Sidebar/components/NetworkDropdown.js:53]: Why the extra else if? `if(userNetworks.length > 0){` @danielmarv

Comment (danielmarv) [reply netmanager/src/views/layouts/common/Sidebar/components/NetworkDropdown.js]: i have resolved the requested changes

**Review by Codebmk — APPROVED** [review]
LGTM @danielmarv Thanks
image

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by FaithDaka — APPROVED** [review]

**Review by Baalmart — APPROVED** [review]
Thanks @danielmarv

**Pull request merged**